[teslascope] Fix warnings

psmedley opened the pull request:
This PR removes (almost) all the warnings when building the teslascope binding.

The remaining warnings are:
```
[WARNING] /home/psmedley/openhab-addons/bundles/org.openhab.binding.teslascope/src/main/java/org/openhab/binding/teslascope/internal/TeslascopeVehicleHandler.java:[137,50] The value of the local variable onOffCommand is not used
[WARNING] /home/psmedley/openhab-addons/bundles/org.openhab.binding.teslascope/src/main/java/org/openhab/binding/teslascope/internal/TeslascopeVehicleHandler.java:[143,50] The value of the local variable onOffCommand is not used
[WARNING] /home/psmedley/openhab-addons/bundles/org.openhab.binding.teslascope/src/main/java/org/openhab/binding/teslascope/internal/TeslascopeVehicleHandler.java:[149,50] The value of the local variable onOffCommand is not used
[WARNING] /home/psmedley/openhab-addons/bundles/org.openhab.binding.teslascope/src/main/java/org/openhab/binding/teslascope/internal/TeslascopeVehicleHandler.java:[155,50] The value of the local variable onOffCommand is not used

```
Which (I think) are safe to ignore.

lsiepel reviewed (changes requested):
Thanks, left a comment. The remaining warnings about the onOffCommand cvan be fixed. Either by removeing the variable assignment from the instanceof condition. Or (even better) check if the onOffCommand is ON before sending the command. For instance the openFrunk() is now called no matter if the command is on or off. I would only expect it to open when set ON.

psmedley commented on bundles/org.openhab.binding.teslascope/src/main/java/org/openhab/binding/teslascope/internal/TeslascopeAccountHandler.java:
```diff
@@ -66,54 +62,64 @@ public ThingUID getUID() {
-            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR,
-                    "Communication problem: " + e.getMessage());
-            return "";
+        TeslascopeAccountConfiguration localConfig = config;
```
done

psmedley commented:
> Thanks, left a comment. The remaining warnings about the onOffCommand cvan be fixed. Either by removeing the variable assignment from the instanceof condition. Or (even better) check if the onOffCommand is ON before sending the command. For instance the openFrunk() is now called no matter if the command is on or off. I would only expect it to open when set ON.

Fixed remaining warnings. I just removed the variable associated with OnOffCommand. The teslascope command is just 'openfrunk' but in cars with a powered frunk, sending this command also closes the frunk.
